proto/frequenz/api/common/v1/metrics/metrics.proto

```diff
@@ -172,6 +172,49 @@ enum Metric {
   METRIC_SENSOR_IRRADIANCE = 166;
 }
 
+// Enumerated categories of connections from which metrics can be obtained for a
+// given electrical component or sensor.
+enum MetricConnectionCategory {
+  // Default value.
+  METRIC_CONNECTION_CATEGORY_UNSPECIFIED = 0;
+
+  // Represents a connection to a battery.
+  METRIC_CONNECTION_CATEGORY_BATTERY = 1;
+
+  // Represents a connection to a PV array.
+  METRIC_CONNECTION_CATEGORY_PV_ARRAY = 2;
```

Comment:
For inverter types we use `SOLAR`, do we really want to use something different here?

`SOLAR` always sounded weird to me though, not sure if it is a industry standard, or if we want to rename the inverter type too, and in that case if it makes sense to use `PV_ARRAY` or just `PV` or avoid acronyms and go for `PHOTOVOLTAIC`?

Reply:
Do you think it is worth changing in the inverter type too?

Reply:
We should. I will create a separate PR for that.